website/content/ko/wgs/platforms/whitepaper/index.md

```diff
@@ -1,505 +1,238 @@
-  </tr>
-  <tr>
-    <td>Identity and secret services</td>
-    <td>Ensure workloads have locators and secrets to use resources and capabilities. Enable services to identify themselves to other services</td>
-    <td>Keycloak, Dex, External Secrets, SPIFFE/SPIRE, Teller, cert-manager</td>
-  </tr>
-  <tr>
-    <td>Security services</td>
-    <td>Observe runtime behavior and report/remediate anomalies. Verify builds and artifacts don't contain vulnerabilities. Constrain activities on the platform per enterprise requirements; notify and/or remediate aberrations</td>
-    <td>Falco, In-toto, KubeArmor, OPA, Kyverno, Cloud Custodian</td>
-  </tr>
-  <tr>
-    <td>Artifact storage </td>
-    <td>Store, publish and secure built artifacts for use in production. Cache and analyze third-party artifacts. Store source code.</td>
-    <td>ArtifactHub, Harbor, Distribution, Porter</td>
-  </tr>
-</table>
-
-<!-- ## Footnotes -->
-
-[1]: https://www.atlassian.com/devops/frameworks/team-topologies
-[2]: https://martinfowler.com/articles/talk-about-platforms.html
-[3]: https://teamtopologies.com/key-concepts-content/what-is-a-thinnest-viable-platform-tvp
-[4]: https://queue.acm.org/detail.cfm?id=3454124
-[5]: https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling
+앞서 설명한 것처럼 클라우드 네이티브 컴퓨팅을 위한 플랫폼은 여러 지원 제공업체의 기능과 서비스를 함께 혼합하여 제공합니다. 이러한 제공업체는 같은 기업 내의 다른 팀일 수도 있고 클라우드 서비스 제공업체와 같은 타사일 수도 있습니다. 간단히 말해, 플랫폼은 기본 기능 제공업체와 애플리케이션 개발자와 같은 플랫폼 사용자를 연결하고, 그 과정에서 보안, 성능, 비용 거버넌스, 일관된 경험에 대해 원하는 모범 사례를 구현하고 시행합니다. 다음 그래픽은 제품, 플랫폼, 기능 제공업체 간의 관계를 보여줍니다.
+
+<figure><img src="https://appdelivery.cncf.io/whitepapers/platforms/assets/platform_components.png" alt=""><figcaption><p>제품과 플랫폼 그리고 기능 제공업체 간의 관계 그래프</p></figcaption></figure>
+
+이 백서에서는 우수한 플랫폼과 플랫폼 팀을 구성하는 방법에 중점을 두었으며, 이제 마지막 섹션에서는 플랫폼이 실제로 제공할 수 있는 기능에 대해 설명하고자 합니다. 이 목록은 플랫폼 구축 담당자를 안내하기 위한 것으로, 일반적으로 클라우드 네이티브 애플리케이션에 필요한 기능을 포함하고 있습니다. 하지만 앞서 언급했듯이 좋은 플랫폼은 사용자의 요구 사항을 반영하므로 궁극적으로 플랫폼 팀은 사용자와 함께 플랫폼이 제공하는 기능을 선택하고 우선 순위를 정해야 합니다.
+
+기능들은 도메인의 특성에 따라 여러가지로 특성 및 속성이 나누어질 수 있습니다. 예를 들어, 관찰 가능성에는 메트릭, 추적 및 로그를 수집 및 게시하고 비용 및 리소스 사용량을 모니터링하기 위한 기능이 포함될 수 있습니다. 이 때 조직에서는 각 기능 또는 요소의 필요성과 우선순위를 고려해서 적용해야 합니다. 추후 CNCF 발행물은 각 도메인에 대해 더 확장해서 배포할 수 있습니다.
+
+다음은 클라우드 네이티브 컴퓨팅을 위한 플랫폼을 구축할 때 고려해야 할 기능 영역입니다.
+
+1. **웹 포털**: 제품 및 기능을 관찰하고 할당 및 배포 관리(provisioning)함
+2. **API(및 CLI)**: 제품 및 기능을 자동으로 할당 및 배포 관리(provisioning)함
+3. **가장 빠른 도입을 위한(Golden Path) 템플릿 및 문서**:  제품에서 기능을 최적으로 사용할 수 있도록 지원
+4. **구축 및 테스트를 위한 자동화**: 서비스 및 제품에 해당
+5. **전달(delivering) 및 검증을 위한 자동화**: 서비스 및 제품에 해당
+6. **개발 환경**:  사용할 수 있도록 설치 구성된(hosted) IDE와 원격 연결 도구 제공
+7. **관찰 가능성**: 계측 및 대시보드를 사용하여 서비스 및 제품을 모니터링(기능, 성능 및 비용 관찰 포함)
+8. **인프라(Infrastructure) 서비스**: 런타임이 동작가능한 컴퓨터, 프로그래밍에 사용할 네트워크, 블록 및 볼륨 스토리지를 제공
+9. **데이터 서비스**: 데이터베이스, 캐시, 객체 저장소, 메시징 및 이벤트 서비스(서비스 브로커, 큐, 이벤트 패브릭을 포함) 제공
+10. **사용자 인증 및 시크릿(Secret) 관리 서비스**: 서비스 및 사용자 ID 및 권한 부여, 인증서 및 키 발급, 정적 시크릿 저장소 제공
```

Comment:
There is **Messaging** sentence in English page, I think you missed it. :)
If you don't mind, I will fix all of them like style, missing sentence. (And I will make a PR to your branch)